### PR TITLE
Add Google Workspace configuration to Kubernetes example

### LINKED ISSUE #204: Google Workspace credentials and settings secrets not correctly available to pods in K8s deployment
Although `op-scim-config.yaml` specifies the workspace credentials as part of the data object:
https://github.com/1Password/scim-examples/blob/ad9c748e9898fd6763ace6b50487553b967a5c6c/kubernetes/op-scim-config.yaml#L18-L19

They do not get mounted in the deployment:
https://github.com/1Password/scim-examples/blob/ad9c748e9898fd6763ace6b50487553b967a5c6c/kubernetes/op-scim-deployment.yaml#L25-L28
https://github.com/1Password/scim-examples/blob/ad9c748e9898fd6763ace6b50487553b967a5c6c/kubernetes/op-scim-deployment.yaml#L39-L42

- [x] Create the reference in `.spec.volumes[]` for each GW secret
- [x] Create the mountpoint in `spec.containers[].volumeMounts[]`

Happy to be the assignee once my repo access is sorted and I'll get this fixed up.

### PR DESCRIPTION
**Pull request by scottisloud**
Previously, the Kubernetes example walked people through the process of creating secrets for Google Workspace credentials, but those secrets were never mounted or used in the deployment. This PR fixes the deployment to support providing Google Workspace credentials at deployment time. It also accommodates a situation in which someone wants to update en existing bridge with Workspace credentials.

The deployment now correctly mounts workspace secrets if they are created. 

* Refactored `volumeMounts` and `volumes` in op-scim-deployment to use a projected volume to accommodate multiple secrets. 
* Workspace secrets are marked as optional so a op-scim-bridge container will be deployed even if these secrets are not present. 
* Updated `kubectl create secret` command to include a key value for Workspace secrets, in line with what has been done for the `scimession` secret previously, to avoid naming discrepancies. 

In my testing, this deployment will work with or without Workspace credentials. It also works to update an existing bridge to have Workspace credentials (an unlikely scenario but 🤷‍♂️ ).

### CODE REVIEW

**Comment by scottisloud** [comment]
closes #204

File: kubernetes/README.md [hunk 1]
```diff
@@ -58,13 +58,13 @@ This section is only relevant to those enrolled in our Google Workspace provisio
...
 ```bash
-kubectl create secret generic workspace-settings --from-file=/path/to/workspace-settings.json
+kubectl create secret generic workspace-settings --from-file=workspace-settings=/path/to/workspace-settings.json
 ```
 
 Then, you will need to provide the Google Service Account key file, also generated according to our documentation. Substitute `<keyfile>` with the filename generated by Google for your Google Service Account.
 
 ```bash
-kubectl create secret generic workspace-credentials --from-file=/path/to/<keyfile>.json
+kubectl create secret generic workspace-credentials --from-file=workspace-credentials=/path/to/<keyfile>.json
```
Comment (ag-adampike) [inline kubernetes/README.md:67]:
```suggestion
kubectl create secret generic workspace-credentials --from-file=workspace-credentials.json=./workspace-credentials.json
// Replace `./workspace-credentials.json` with `[path/]filename.json` if saved elsewhere or with a different filename
```

File: kubernetes/op-scim-deployment.yaml [hunk 2]
```diff
@@ -23,7 +23,7 @@ spec:
             - name: http
               containerPort: 3002
           volumeMounts:
-            - name: scimsession
+            - name: secrets
```
Comment (ag-adampike) [inline kubernetes/op-scim-deployment.yaml:26]:
```suggestion
            - name: secret
```

Given the value for this is an arbitrary label, perhaps we should consider changing this to match the directory name.

File: kubernetes/op-scim-deployment.yaml [hunk 3]
```diff
@@ -37,6 +37,14 @@ spec:
             - configMapRef:
                 name: op-scim-configmap
       volumes:
-        - name: scimsession
-          secret:
-            secretName: scimsession
+      - name: secrets
```
Comment (ag-adampike) [inline kubernetes/op-scim-deployment.yaml:40]:
```suggestion
      - name: secret
```

**Review by ag-adampike — CHANGES_REQUESTED** [review]
This is great work, but needs one small change to the `kubectl create secret` commands in order for the Secrets to be mounted with the `.json` extension intact.

File: kubernetes/README.md [hunk 4]
```diff
@@ -58,13 +58,13 @@ This section is only relevant to those enrolled in our Google Workspace provisio
 First, you’ll need to edit the file located at `scim-examples/beta/workspace-settings.json`, and fill in the values according to those presented in our Google Workspace documentation.
 
 ```bash
-kubectl create secret generic workspace-settings --from-file=/path/to/workspace-settings.json
+kubectl create secret generic workspace-settings --from-file=workspace-settings=/path/to/workspace-settings.json
```
[the remark this answers is not in the document]
Comment (scottisloud) [reply kubernetes/README.md:61]: .json is not needed on the keyname in my testing.

**Comment by scottisloud** [comment]
Integrated and tested Pike's suggestions and updated the readme with a brief note about potential ignorable errors a person may get when storing their workspace json files in the same directory as the deployment.

**Review by ag-adampike — DISMISSED** [review]
LGTM!!:tada:

**Comment by scottisloud** [comment]
Apparently Git wasn't signing my commits 😫 
Working on figuring out why that's the case.

**Comment by scottisloud** [comment]
Phew, after a harrowing rebase I've managed to get all previous commits signed.

**Review by ag-adampike — APPROVED** [review]

**Comment by black-bryan** [comment]
Successfully tested K8 deployment with workspace-settings and workspace-credentials supplied as K8 secrets, as well as non GW bridge deployment.

**Review by chasdl — APPROVED** [review]
These changes look solid to me, appreciate the effort here!

**Pull request merged**